l10n_it_corrispettivi/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 {
     'name': 'Italian Localization - Corrispettivi',
-    'version': '10.0.1.2.2',
+    'version': '10.0.1.3.0',
```

Comment:
same as https://github.com/OCA/l10n-italy/pull/1157#discussion_r272043749

Reply:
Thanks @tafaRU for the review!
Done